bdk_chain/src/tx_graph.rs

```diff
@@ -22,20 +28,17 @@ impl Default for TxNode {
 }
 
 impl TxGraph {
+    /// The total number of transactions (full and partial) that exists in the [`TxGraph`].
+    pub fn count(&self) -> usize {
+        self.txs.len()
+    }
```

Comment:
Why do we need this?

Reply:
Maybe we can just not have a size hint for now.

The correct solution to this is to store full txs and outputs in different maps so you can easily count how many you have of each. These seems out of scope for this PR so I'd remove this and the size hint.